### PR TITLE
Fix test_kms_with_rest errors\

### PR DESCRIPTION
**Pull request by HawieLong**
1. fix the upper used in test_kms_with_rest which cause base64 error
2. fix disable key coding error

Signed-off-by: Long Hanyu <hanyu.long@intel.com>

### CODE REVIEW

File: ehsm_kms_service/test/test_kms_with_rest.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def test_Stest_RSA3072_sign_verify(base_url, headers):
 
     # test Sign
     payload.clear()
-    payload["digest"] = str(base64.b64encode("test".encode("utf-8")),'utf-8').upper()
+    payload["digest"] = str(base64.b64encode("test".encode("utf-8")),'utf-8')
```
Comment (yang8621) [inline ehsm_kms_service/test/test_kms_with_rest.py:180]: nice fix. 
but still same issue @line 217 229  255  267

**Comment by yang8621** [comment]
Please also fix: https://github.com/intel/ehsm/blob/main/ehsm_kms_service/test/cli/_utils_.py#L28

File: ehsm_kms_service/test/test_kms_with_rest.py (hunk 1 above)
Comment (HawieLong) [reply ehsm_kms_service/test/test_kms_with_rest.py:180]: the sign will fix later, since the sign is aligned with server, the sever is using upper now.

Comment (yang8621) [reply ehsm_kms_service/test/test_kms_with_rest.py:180]: Seems: https://github.com/intel/ehsm/blob/main/ehsm_kms_service/function.js#L603

**Review by yang8621 — APPROVED** [review]

**Review by syan10 — APPROVED** [review]

**Pull request merged**